cloud-controller-manager/do/metadata.go

```diff
@@ -17,24 +17,49 @@ limitations under the License.
 package do
 
 import (
+	"context"
 	"fmt"
 	"io/ioutil"
 	"net/http"
 	"os"
 
+	"github.com/digitalocean/godo"
+	"github.com/pkg/errors"
 	"k8s.io/klog/v2"
 )
 
 const dropletRegionMetadataURL = "http://169.254.169.254/metadata/v1/region"
 
 // dropletRegion returns the region of the currently running program.
 func dropletRegion() (string, error) {
-	// FAKE_REGION can be specified for local testing.
-	if region := os.Getenv("FAKE_REGION"); region != "" {
-		klog.Warningf("Using fake region %q from environment variable FAKE_REGION -- this should only be set for testing purposes!", region)
-		return region, nil
+	// REGION can be specified for local testing.
+	region := os.Getenv("REGION")
+	if region == "" {
+		return httpGet(dropletRegionMetadataURL)
 	}
-	return httpGet(dropletRegionMetadataURL)
+
+	if !validRegion(region) {
```

Comment:
Could we also document in the README that the DO Regions service will be used to validate an explicitly specified region? This may be helpful for users that tighten network access for security reasons.